.gitlab-ci.yml

```diff
@@ -43,37 +43,37 @@ stages:
 unit-tests:python-2.7:
   <<: *unit-tests
   variables:
-    PYTHON_VERSION: '2.7'
+    PYTHON_VERSION: "2.7"
```

Comment:
Where do these formatting changes to the Yaml file come from? Is there now a linter for that? (Where?)

Reply:
I think they automatically got applied when saving the file because the prettier plugin of code kicked in.